app/views/companies/new.html.erb

```diff
@@ -48,6 +43,16 @@
     </div>
   </div>
 
+  <div class="form-group">
```

Comment:
This was dan's idea - makes sense since we have the relationships designed to be one to many. this way, we can accommodate that in the form.